server/models/Notification.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  user: {
```

Comment:
This field keeps track of the recipient. How do we keep track of the sender/creator?